pages/about.tsx

```diff
@@ -1,15 +1,55 @@
-import Link from 'next/link'
 import Layout from '../components/Layout'
+import OfficerIcon from '../components/OfficerIcon'
+import OfficerJson from '../data/Officer.json'
+
+var presEmail = "";
+var presName = "";
```

Comment:
I think instead of having these vars here it'd be better to use nextjs' static generation functionality to get the email and name value for the president and then pass it in as props (we also would probably want to pass in the OfficerJson as props as well) - here's a good example of how to do it: https://nextjs.org/docs/basic-features/pages#static-generation-recommended (The example is under the heading `Static Generation with data`)

You'll probably have to do a for loop in that `getStaticProps()` method to get the officer name and email (which you can then pass in as props -> You're going to pass in OfficerJson and President (or whatever you want to call the object) as props) which might not seem great since we already have the `OfficerJson.map` loop to render the officer cards/images but if this were in prod that forEach loop would only run one time - when we deploy the site and that is due to it being a static page

If you need any help with this or if what I said doesn't make sense just let me know - understanding next and the static vs serverside vs client side rendering stuff/functions can be confusing